src/common/models/customization/customization.ts

```diff
@@ -0,0 +1,11 @@
+export interface Customization {
+  headerBackground?: string;
+  svgIcon?: string;
```

Comment:
I would call this `customLogoSvg` because it is "the custom logo svg that will be rendered"